lib/pure/os.nim

```diff
@@ -2341,46 +2347,47 @@ proc copyDir*(source, dest: string) {.rtl, extern: "nos$1",
   ##
   ## On other platforms created files and directories will inherit the
   ## default permissions of a newly created file/directory for the user.
-  ## Use `copyDirWithPermissions proc <#copyDirWithPermissions,string,string>`_
+  ## Use `copyDirWithPermissions proc <#copyDirWithPermissions,string,string,bool>`_
   ## to preserve attributes recursively on these platforms.
   ##
   ## See also:
-  ## * `copyDirWithPermissions proc <#copyDirWithPermissions,string,string>`_
-  ## * `copyFile proc <#copyFile,string,string>`_
+  ## * `copyDirWithPermissions proc <#copyDirWithPermissions,string,string,bool>`_
+  ## * `copyFile proc <#copyFile,string,string,bool>`_
   ## * `copyFileWithPermissions proc <#copyFileWithPermissions,string,string>`_
   ## * `removeDir proc <#removeDir,string>`_
   ## * `existsOrCreateDir proc <#existsOrCreateDir,string>`_
   ## * `createDir proc <#createDir,string>`_
-  ## * `moveDir proc <#moveDir,string,string>`_
+  ## * `moveDir proc <#moveDir,string,string,bool>`_
   createDir(dest)
   for kind, path in walkDir(source):
     var noSource = splitPath(path).tail
     case kind
     of pcFile:
-      copyFile(path, dest / noSource)
+      copyFile(path, dest / noSource, followSymlinks = followSymlinks)
```

Comment:
ugh.. looks like a pre-existing bug: symlinks are not copied?? that means your code here won't really make a difference because path is a pcFile, not symlink

Reply:
IMO `followSymlinks = true` makes no sense for copyDir, it should just have the meaning `followSymlinks = false`, ie remove the param `followSymlinks`:

```nim
    case kind
    of pcDir:
      copyDir(path, dest / noSource, followSymlinks = false)
    else:
      copyFile(path, dest / noSource, followSymlinks = false)
```

Reply:
See: https://github.com/nim-lang/Nim/pull/16709#issuecomment-761132187